src/auth/authApp.js

```diff
@@ -166,12 +134,12 @@ export function signUserOut(redirectURL: ?string = null) { // eslint-disable-lin
  * by special authenticators.
  * @return {String} the authentication request
  */
-export function makeAuthRequest(transitPrivateKey: string,
-                                redirectURI: string,
-                                manifestURI: string,
-                                scopes: Array<string>,
+export function makeAuthRequest(transitPrivateKey: ?string,
+                                redirectURI: string = `${window.location.origin}/`, 
```

Comment:
These defaults make this function un-callable from non-browser contexts.

Reply:
Actually, nevermind -- it seems like `makeAuthRequest` is deprecated anyways, and non-browser apps would use the UserSession API anyways.